doc/tutorial/getting-started/laminar-scalablytyped.md

```diff
@@ -0,0 +1,808 @@
+However, if we put form fields in the rows, the experience will significantly degrade.
+
+We would like to *reuse* the `tr` elements for the `DataItem`s that were already rendered before.
+Ideally, for any given `id: DataItemID`, we would like the same rendered `tr` to be reused.
+
+This is exactly what Laminar's `split` method provides.
+Let us amend the binding for `children` as follows:
+
+{% highlight scala %}
+children <-- dataSignal.split(_.id) { (id, initial, itemSignal) =>
+  renderDataItem(id, itemSignal)
+},
+{% endhighlight %}
+
+`split` can be applied on a `Signal[List[T]]` and returns a `Signal[List[U]]`.
+Whereas `map` would systematically create new output values, `split` reuses the output values for the inputs whose `_.id` is the same as before.
+In other words, `_.id` is the *key* used by `split` to find previously rendered elements.
+
+For input elements whose key (or ID) existed before, it reuses a previously computed element.
+For new input elements, with an unknown key, it calls the 3-parameter function with the following arguments:
+
+1. `id: DataItemID`: the key of the newly found input element.
+1. `initial: DataItem`: the input element with that ID, as found the first time.
+1. `itemSignal: Signal[DataItem]`: a signal of all the `DataItem`s that will be found with the same `id` in the future, starting with `initial` itself.
+
+The `id` is the only field taken into account to reuse an output element.
+Therefore, if a future input contains a different `DataItem` with the same `id`, it will return the same output element.
+That `DataItem` will then become the new value of the time-varying `itemSignal`.
+This is why we now pass `itemSignal` to `renderDataItem`, which we amend to accept a `Signal[DataItem]`:
+
+{% highlight diff %}
+-  def renderDataItem(id: DataItemID, item: DataItem): Element =
++  def renderDataItem(id: DataItemID, itemSignal: Signal[DataItem]): Element =
+     tr(
+-      td(item.label),
+-      td(item.value),
++      td(child.text <-- itemSignal.map(_.label)),
++      td(child.text <-- itemSignal.map(_.value)),
+       td(button("🗑️", onClick --> (_ => removeDataItem(id)))),
+     )
+   end renderDataItem
+{% endhighlight %}
+
+We cannot directly read `item.label` or `item.value` anymore.
+Instead, we use `map` to get time-varying views of the label and value, which we bind to the text of the `td` elements using `child.text <--`.
+You may recall that we had done something very similar in our initial `counter` example:
+
+{% highlight scala %}
+    button(
+      ...
+      "count is ",
+      child.text <-- counter,
+      ...
+    )
+{% endhighlight %}
+
+We now have optimal reuse of `tr` elements based on the `id` of the input `DataItem`s.
+
+[See it in action and fiddle with the code in Scribble](https://scribble.ninja/u/sjrd/cwhugissosbhkucvskpwbqedrntp)
+
+## The actual chart
```

Comment:
This could be a third page of the tutorial, as it solves an independent problem using an independent set of tools.